fastfood/food.py

```diff
@@ -141,85 +151,29 @@ def update_cookbook(cookbook, templatepack, stencilset_name, **options):
     return written_files, cookbook
 
 
-def create_new_cookbook(cookbook_name, templatepack,
-                        cookbooks_home, force=False):
+def create_new_cookbook(cookbook_name, cookbooks_home, force=False):
```

Comment:
Force is actually still used in the `process_stencil` method. I'll remove it from this method signature, though. I'm not sure 'force' has much meaning now that `create_new_cookbook` is basically a fancy `mkdir`.